cypress/integration/errorPageNews.js

```diff
@@ -19,6 +20,10 @@ describe('Article Body Tests', () => {
     testResponseCode(`/news/articles/${config.assets.nonExistent}`, 404);
   });
 
+  it('should have the correct lang & dir attributes', () => {
+    hasHtmlLangDirAttributes({ lang: 'en_GB', dir: 'ltr' });
```

Comment:
This will fail on test 🙈 unless we update mozart pre merge

Reply:
Thanks for raising this. I often forget that the Mozart static error pages are used on Test and Live. I've limited this test for now, to be removed after this issue is complete: https://github.com/bbc/simorgh-infrastructure/issues/347